frontend/src/components/Container/index.tsx

```diff
@@ -75,6 +84,10 @@ const FormContainer = () => {
     const payload = {
       verification_code: data.verificationCode,
     }
+    setState({
+      ...state,
```

Comment:
You don't need to ... The state, it automatically does that. Just return the keys you want to modify.

Reply:
Ah okay. I'll sort this out tonight